.github/workflows/ci-waspc-build.yaml

```diff
@@ -22,6 +22,7 @@ env:
 
 jobs:
   build:
+    name: Build Wasp
```

Comment:
This is wrong, we are already defining a name on L90 https://github.com/wasp-lang/wasp/blob/main/.github/workflows/ci-waspc-build.yaml#L90